package-lock.json

```diff
@@ -18976,6 +18976,14 @@
         "tryer": "^1.0.0"
       }
     },
+    "bidi-js": {
```

Comment:
I think we only want the troika related packages here. I feel that some things have leaked.